api-2.1/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ObservationTranslatorImpl_2_1.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.translators.impl;
```

Comment:
I don't *think* we actually need a custom version of the `ObservationTranslator` here, since the default version still calls the `ObservationStatusTranslator`. We just need to ensure that is properly overridden.

Reply:
hey Ian, this might work with status but what about translating the interpretations and it's not part of the origin implementation with platform 2.0.5

Reply:
I think the solution there is to add an translator for interpretations similar to the dummy translator for observation status we have. That way, it's easy for implementors to support interpretations if they're currently capturing that information.

Reply:
That makes sense, I'll look into it